game/src/contrib/components/XPComponent.java

```diff
@@ -3,126 +3,177 @@
 import core.Component;
 import core.Entity;
 
-import java.util.function.LongConsumer;
-
-public class XPComponent extends Component {
-
-    private static final double LEVEL_1_XP = 100;
+import java.util.function.Consumer;
+
+/**
+ * Allow associated entity to collect Experience Points (XP) and level up if enough XP is collected.
+ *
+ * <p>The component stores the current character level, the amount of XP this entity has collected,
```

Comment:
Wie wird collected: Aufsummieren aller XP vom Start an oder wird beim Aufleveln die XP wieder auf 0 gesetzt?

Reply:
Aufsummiert.
Habs in die doc aufgenommen-